scripts/docker-startup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
```

Comment:
I think we should add short description what is this script about, like:

`# This script is used internally by Docker development environment, you can read more about it in DOCKER.md.`